lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -245,6 +259,26 @@ class MetroItinerary<Props> extends NarrativeItinerary {
       currency
     )
 
+    const co2VsBaseline = Math.round(itinerary.co2VsBaseline * 100)
+    const emissionsNote = !mini && co2VsBaseline < -20 && (
```

Comment:
Is this 20 related to the threshold we saw previously?

Reply:
uhhh maybe. it's gone now, uses the config!